<ui version="4.0" >
 <class>LineUi</class>
 <widget class="QWidget" name="LineUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" stdset="0" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="OffsetValueL" >
     <property name="text" >
      <string>Of&amp;fset:</string>
     </property>
     <property name="buddy" >
      <cstring>OffsetLE</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="OffsetLE" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>Value of the vertical line offset.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="lyx::frontend::LengthCombo" name="OffsetUnitCO" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="WidthValueL" >
     <property name="text" >
      <string>&amp;Width:</string>
     </property>
     <property name="buddy" >
      <cstring>WidthLE</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="WidthLE" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>Value of the line width.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="lyx::frontend::LengthCombo" name="WidthUnitCO" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="HeightValueL" >
     <property name="text" >
      <string>&amp;Thickness:</string>
     </property>
     <property name="buddy" >
      <cstring>HeightLE</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QLineEdit" name="HeightLE" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="toolTip" >
      <string>Value of the line thickness.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" >
    <widget class="lyx::frontend::LengthCombo" name="HeightUnitCO" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>150</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lyx::frontend::LengthCombo</class>
   <extends>QComboBox</extends>
   <header>LengthCombo.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>WidthLE</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
